go.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/hashicorp/go-hclog v0.14.1
 	github.com/hashicorp/go-memdb v1.3.2
 	github.com/hashicorp/go-multierror v1.1.0
-	github.com/hashicorp/go-plugin v1.3.0
+	github.com/hashicorp/go-plugin v1.4.2
```

Comment:
Upgrading to get the ProtocolVersion field on ReattachConfig, which we need to know which VersionedPlugins to use.